src/main/java/io/jenkins/plugins/checks/github/CheckRunGHEventSubscriber.java

```diff
@@ -93,6 +88,7 @@ protected void onEvent(final GHSubscriberEvent event) {
 
             LOGGER.log(Level.INFO, "Received rerun request through GitHub checks API.");
             try (ACLContext ignored = ACL.as(ACL.SYSTEM)) {
+                String branchName = payloadJSON.getJSONObject("check_run").getJSONObject("check_suite").optString("head_branch");
```

Comment:
Is `getJSONObject("check_suite")` always available or do we need a guard here is well?

Reply:
it's always available